modules/build/src/main/scala/scala/build/input/Inputs.scala

```diff
@@ -263,7 +263,8 @@ object Inputs {
     stdinOpt: => Option[Array[Byte]],
     acceptFds: Boolean,
     enableMarkdown: Boolean,
-    isRunWithShebang: Boolean
+    isRunWithShebang: Boolean,
+    isRunWithDefault: Boolean
```

Comment:
it might be a better idea to handle this by an `enum` rather than a myriad of boolean flags
```suggestion
    invokingCommand: ScalaCliCommandRef
```